---
geometry_msgs/PoseArray waypoints
string[] names
